### ISSUE TITLE
[Accessibility]A11y_.NET Online REPL View_Keyboard: Focus is not properly visible on "Enter focus mode" and "Run" buttons

### ISSUE THREAD

**Issue by soumyatippapuram** [dotnet/try#104]
**Environment Details:** 
URL: https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=1
Browser: Chrome 67.0.3396.87

**Repro Steps:**
1)Hit the URL : https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=1
2)Tab till "Enter focus mode" button and press enter
3)Check if the focus is properly visible on "Enter focus mode" and "Run" buttons 

**Actual Result:**
Focus is not properly visible on "Enter focus mode" and "Run" buttons

**Expected Result:**
Focus should be properly visible on "Enter focus mode" and "Run" buttons

**User Impact:** 
Keyboard only users may not know which element has the current focus
![bug_enter focus mode_focus not visible](https://user-images.githubusercontent.com/41108985/42629235-54afd0c0-85f0-11e8-9442-f98b1c33e762.png)
![bug_run_focus not visible](https://user-images.githubusercontent.com/41108985/42629236-54da26e0-85f0-11e8-8146-883ff5ab49e9.png)

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Comment by LadyNaggaga** [comment]
@dend  @mairaw  @BillWagner  Is this an accessibility bug your team can address? This does fall out the Try .NET editor. Thank you

**Comment by LadyNaggaga** [comment]
https://github.com/MicrosoftDocs/feedback/issues/735 issue has been resolved.

**Comment by umasarath52** [comment]
This was discussed with Ryan and approved to close as this is non blocking scenario.
